### ISSUE TITLE
as requested:  Support for GCE (but I'd rather support for bare metal!)

### ISSUE THREAD

**Issue by faddat** [kontena/kontena#375]
;)

GCE is my favorite public cloud though.

**Comment by miskun** [comment]
Yes, let's add GCE support. I think we have working prototype somewhere, just need some time to polish.

**Comment by jnummelin** [comment]
Should be now fairly simple to implement this as a plugin since we have the plugin "framework" in place.
